trivia/ui.html

```diff
@@ -0,0 +1,21 @@
+<script>
+  window.onmessage = async (event) => {
+    const msg = event.data.pluginMessage;
+    if (msg.type === "category") {
+      const res = await fetch("https://opentdb.com/api_category.php");
```

Comment:
Minor: it could be cleaner to just send the URL for this query over as well. Then just have the `ui.html` only take a url parameter and send back the fetched JSON.